tests/system/src/testscriptapp1/java/mymod/myapp1/FXMLScriptTest.java

```diff
@@ -0,0 +1,303 @@
+        }
+        catch (Exception ioe)
+        {
+            ioe.printStackTrace();
+            System.exit(-1);
+        }
+            // fire three events on the button
+        btn.fire();
+        btn.fireEvent( new ActionEvent() );
+        btn.fireEvent( new MouseEvent( MouseEvent.MOUSE_CLICKED,
+                                       0,       // double x,
+                                       0,       // double y,
+                                       0,       // double screenX,
+                                       0,       // double screenY,
+                                       MouseButton.PRIMARY,         // MouseButton button,
+                                       0,       // int clickCount,
+                                       false,   // boolean shiftDown,
+                                       false,   // boolean controlDown,
+                                       false,   // boolean altDown,
+                                       false,   // boolean metaDown,
+                                       true,    // boolean primaryButtonDown,
+                                       false,   // boolean middleButtonDown,
+                                       false,   // boolean secondaryButtonDown,
+                                       false,   // boolean synthesized,
+                                       false,   // boolean popupTrigger,
+                                       false,   // boolean stillSincePress,
+                                       null     // PickResult pickResult
+                                     )
+                      );
+
+        // mainStage.setScene(scene);
+        // mainStage.show();
+        Platform.exit();
+    }
+
+    // show engine invocations with script text and their Bindings
+    static void dumpEvalInformation()
+    {
+        System.err.println("\nListing eval() invocation information (invocationList):");
+
+        Iterator<RgfPseudoScriptEngine> it = RgfPseudoScriptEngine.getEnginesUsed().iterator();
+        while (it.hasNext())
+        {
+            RgfPseudoScriptEngine rpse = it.next();
+            ArrayList invocationList   = rpse.getInvocationList();
+            System.err.println("ScriptEngine: ["+rpse+"]");
+
+            Iterator<InvocationInfos> itEval = invocationList.iterator();
+            int count = 1;
+            while (itEval.hasNext())
+            {
+                System.err.println("\teval() invocation # "+count+": ");
+                InvocationInfos entry = itEval.next();
+                System.err.println(entry.toDebugFormat("\t\t"));    // indentation
+                count++;
+                System.err.println();
+            }
+        }
+    }
+
+    static void assertCorrectInvocations()
+    {
+            // test only creates one engine for a script controller
+        Util.assertTrue( "exactly one pseudo script engine instance", RgfPseudoScriptEngine.getEnginesUsed().size()==1);
```

Comment:
I recommend to wrap this line. Also the spacing isn't quite right (see earlier comments).